data/random-teams.ts

```diff
@@ -277,6 +280,53 @@ export class RandomTeams {
 		return samples;
 	}
 
+	/**
+	 * Check if user has directly tried to ban/unban/restrict things in a custom battle.
+	 * Doesn't count bans nested inside other formats/rules.
+	 */
+	private hasDirectCustomBanlistChanges() {
+		if (!this.format.customRules) return false;
+		for (const rule of this.format.customRules) {
+			for (const banlistOperator of ['-', '+', '*']) {
+				if (rule.startsWith(banlistOperator)) return true;
+			}
+		}
+		return false;
+	}
+
+	/**
+	 * Inform user when custom bans are unsupported in a team generator.
+	 */
+	protected enforceNoDirectCustomBanlistChanges() {
+		if (!this.hasDirectCustomBanlistChanges()) return;
+		throw new Error(`Custom bans are not currently supported in ${this.format.name}.`);
```

Comment:
```suggestion
		if (this.hasDirectCustomBanlistChanges()) {
			throw new Error(`Custom bans are not currently supported in ${this.format.name}.`);
		}
```
Typically PS code formats these kinds checks like so; you don't need to rewrite the code that's already here (unless you want to), but next time you contribute this would be good to remember.

Reply:
Thanks, I'll try to keep that in mind.